### DISCUSSION TITLE
Jira Alert - Missing link?

### DISCUSSION THREAD

**Discussion by Hugefarsen** [jertel/elastalert2#543] [Q&A] [answered]
Hello,

When looking at the documentation of Alerts and Jira it says that there will be a link provided to the ticket in question, however I cant find it.

> ### Jira
> The Jira alerter will open a ticket on Jira whenever an alert is triggered. You must have a service account for ElastAlert 2 to connect with. The credentials of the service account are loaded from a separate file. **The ticket number will be written to the alert pipeline**, and if it is followed by an email alerter, a link will be included in the email.

What I would like to do is to pipe that ticket, to Slack so there is a clickable link in my message, is that possible?

**Comment by nsano-rururu** [comment, marked as answer]
I don't think I can.

**Reply by nsano-rururu** [reply]
We recommend looking for tools that are not elastalert2.

**Reply by jertel** [reply]
To add more clarity to this, only the email alerter consumes the JIRA pipeline data today. If you would like to submit a PR to enable the same functionality on the Slack alerter that would be great. See the code on how the emailer alerter accesses the pipeline data here: https://github.com/jertel/elastalert2/blob/b075b47c655767c86ab2c40904960ddb896c8883/elastalert/alerters/email.py#L52

**Reply by Hugefarsen** [reply]
@jertel https://github.com/jertel/elastalert2/pull/547 :)
